### PR TITLE
series0ne/corda-demobench-node-config-fix

### PR DESCRIPTION
**Pull request by series0ne**
Fixes an issue where profiles don't load because the node.conf format has changed between V1 and V3
@chrisr3 please can you review as per @roger3cev 's suggestion. Thanks.

### CODE REVIEW

File: tools/demobench/src/main/kotlin/net/corda/demobench/model/NodeRpcSettings.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package net.corda.demobench.model
+
```
[the remark this answers is not in the document]
Comment (series0ne) [reply tools/demobench/src/main/kotlin/net/corda/demobench/model/NodeRpcSettings.kt]: Yes, in the interest of applying DRY, you/we should be using an existing class, I just couldn’t find one suitable at the time.

File: tools/demobench/src/test/kotlin/net/corda/demobench/model/NodeConfigTest.kt [hunk 2]
```diff
@@ -21,21 +24,21 @@ class NodeConfigTest {
     @Test
```
[the remark this answers is not in the document]
Comment (series0ne) [reply tools/demobench/src/test/kotlin/net/corda/demobench/model/NodeConfigTest.kt]: I would imagine IntelliJ auto-formatting has something to do with this; though I would argue that a scrupulous application of CTRL+ALT+L on source files is usually a good thing (for consistency purposes), which probably wasn’t applied before I got my hands on this file.

**Comment by joeldudleyr3** [comment]
@series0ne Hey Matthew, this is great. Can you also state that the PR is in accordance with the Developer's Certificate of Origin here: https://github.com/corda/corda/blob/master/CONTRIBUTING.md#developer-certificate-of-origin?

**Comment by series0ne** [comment]
My contribution is in accordance with the Developer’s Certificate of Origin.

**Comment by series0ne** [comment]
@joeldudleyr3 @mikehearn 

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

**Comment by mikehearn** [comment]
Yep, thanks!

**Pull request merged**